pyfr/solvers/baseadvec/elements.py

```diff
@@ -99,3 +98,25 @@ def filter_soln(uin):
                 return self._be.ordered_meta_kernel([mul, copy])
 
             kernels['filter_soln'] = filter_soln
+
+        shock_capturing = self.cfg.get('solver', 'shock-capturing', 'none')
+        if shock_capturing == 'entropy-filter':
+            tags = {'align'}
+
+            # Allocate one minimum entropy value per interface
+            self.nfaces = len(self.nfacefpts)
+            self.entmin_int = self._be.matrix((self.nfaces, self.neles),
+                                               tags=tags, extent='entmin_int')
```

Comment:
Don't forget to include the nonce in the extent name.

Reply:
I thought nonce was only necessary when using multi-p?

Reply:
It is needed in general to enable multiple independent instances of a system to co-exist.  Multi-p is a good source of these, but they also arise in testing, too.